sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2929,6 +2929,27 @@ pi_result hip_piEnqueueKernelLaunch(
       retImplEv->start();
     }
 
+    // Set local mem max size if env var is present
+    static const char *local_mem_sz_ptr =
+        std::getenv("SYCL_PI_HIP_MAX_LOCAL_MEM_SIZE");
```

Comment:
Could you please add documentation for this environment variable to sycl/doc.

Reply:
I added it to sycl/doc. Thanks.

Reply:
Thanks!